src/react/components/RichTextEditor.jsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import {Editor, EditorState, ContentState, RichUtils, getDefaultKeyBinding, Modifier, SelectionState} from 'draft-js';
+import htmlToDraft from 'html-to-draftjs';
+import './TextEditor/RichText.css';
+import 'draft-js/dist/Draft.css';
+
+import { InlineStyleControls, BlockStyleControls } from './TextEditor/';
+
+      class RichEditor extends React.Component {
```

Comment:
Intendation is funky here + could this be migrated to a function component for consistency (all the others are)?

Reply:
Also everything else is using named exports so lets continue doing that in here as well instead of default export.